src/docker/DockerApi.ts

```diff
@@ -301,16 +298,15 @@ class DockerApi {
     ) {
         const self = this
 
-        const nameAndTag = imageNameIncludingTag.split(':')
-
-        const tag = nameAndTag[1] || 'latest'
-        const imageName = nameAndTag[0]
+        const { repository, tag } = dockerodeUtils.parseRepositoryTag(
+            imageNameIncludingTag
+        )
 
         return Promise.resolve()
             .then(function() {
                 return self.dockerode.createImage({
-                    fromImage: imageName,
-                    tag: tag,
+                    fromImage: repository,
+                    tag: tag || 'latest',
```

Comment:
I'll move this default case up to make sure that if `tag` gets used anywhere else in this method, it's got the right value.